src/Homepage.tsx

```diff
@@ -1,7 +1,9 @@
 import React from 'react'
 
 import '@patternfly/react-core/dist/styles/base.css'
-import { Page, PageHeader } from '@patternfly/react-core'
+import { Page, PageHeader, PageSection } from '@patternfly/react-core'
+
+import SearchBar from './components/SearchBar'
```

Comment:
css file name should match with component filename

Reply:
Not using any CSS for `Homepage.tsx`. This line is importing the `SearchBar` component. The CSS file for that component is already named `SearchBar.css`